superdesk/storage/mimetype_mixin.py

```diff
@@ -0,0 +1,46 @@
+import io
+import logging
+import mimetypes
+
+import magic
+from werkzeug import datastructures
+
+
+logger = logging.getLogger(__name__)
+
+
+class MimetypeMixin:
```

Comment:
just wondering why mixing and not just a function?

Reply:
`get_mimetype` method is very specific to `SuperdeskGridFSMediaStorage` and `AmazonMediaStorage`. IMO the best place for it is `eve.io.media.MediaStorage`, but since it's not an option I've used mixin for it. Having a function might make somebody think that it's reusable, but it's not. It has very specific args. 
I can move it to `superdesk/media/media_operations.py` as a function if you think it's a better place.